spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -1166,7 +1151,7 @@ public synchronized void testDeleteWithSnapshotIsolation()
 
               for (int numOperations = 0; numOperations < 20; numOperations++) {
                 while (shouldAppend.get() && barrier.get() < numOperations * 2) {
-                  sleep(10);
+                  sleep(200);
```

Comment:
is this being flaky and requires a bump in wait time? We might want to update this to use `Awaitility` in a follow-up PR

Reply:
thanks! yes, the test seems to be flaky. After the updates, the test sometimes failed and sometimes succeeded. I was investigating the issue, but let me update the part with `Awaitlity`.